.golangci-kal.yml

```diff
@@ -96,11 +96,11 @@ linters:
       linters:
         - kubeapilinter
     - path: "api/addons/v1beta2|api/bootstrap/kubeadm/v1beta2|api/controlplane/kubeadm/v1beta2|api/core/v1beta2|api/ipam/v1beta2|api/runtime/v1beta2"
-      text: "ssatags: Conditions should have a listType marker for proper Server-Side Apply behavior"
+      text: "ssatags: .*Conditions should have a listType marker for proper Server-Side Apply behavior"
```

Comment:
Just as an aside, why didn't we add list type atomic to these? Because it conflicts with the expectations of the conditions linter I expect?